Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -1683,7 +1688,7 @@ def get_host_list_detection(self, since_datetime, next_page=None) -> Union[str,
             url_suffix=urljoin(API_SUFFIX, 'asset/host/vm/detection/?action=list'),
             resp_type='text',
             params=params,
-            timeout=60,
+            timeout=240,
```

Comment:
this feels too much no? after we receive this data we still need to send it to XSIAM in batches and this might take some time which will surpass the 300 seconds fetch assets is limited to

Reply:
@moishce I see you resolved this without any comment or change. Please check.